RedditRadio.js

```diff
@@ -240,7 +240,8 @@ class RedditRadio
 			} else {
 				console.log("Built-in command from offline member: " + cmdID);
 			}
-			this[cmdName].apply(this, [ msg ].concat(parse.slice(1)));
+			parse.shift();
+			this[cmdName].apply(this, [ msg ].concat(parse.join(' ')));
```

Comment:
Why?

Reply:
If I left it like it was before, instead of full name it would look up for first word only.

Reply:
This breaks the command system. Instead, you should use something like [this](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Functions/arguments) in the actual command handler instead.

Reply:
I don't really understand the way you implemented commands, so if you can, you can change it on your own. I get the idea of `arguments[]`, tho.

Reply:
Unless you mean `parse` is used later, then it may in fact be problem.